prisma/seed.ts

```diff
@@ -0,0 +1,87 @@
+import prisma from '../prisma/index'; // Relative path required, path aliases throw error with seed command
```

Comment:
Path aliases don't seem to work with the seed command so I had to use relative paths. I think it has to do with the fact that the seed command is run using `ts-node` and we'd need to use something like [tsconfig-paths](tsconfig-paths) to get path aliases to work with it

Reply:
Good note